README.md

```diff
@@ -467,6 +467,9 @@ values stored in `argv._`).  Set `desc` to `false` to create a hidden command.
 Hidden commands don't show up in the help output and aren't available for
 completion.
 
+Note that commands will **not** automatically inherit the configuration of their
```

Comment:
@bcoe Did you mean `...only [global](#global) `**`options`**` are...`?

I like this approach, though I think mentioning inheritance of parent options (if you meant that) is only half of the story, since this PR is also heading to inform people that commands need to be reconfigured and don't just use the configuration of their parent context, such as `.strict()` needs to be re-applied to every sub command.

So let's combine informing about inheritance of the configuration, and the inheritance of options (also mentioning the global method), like this:

```
Note that commands will not automatically inherit _configuration or options_ of their parent context. 
This means you'll have to re-apply configuration if necessary, and make options global manually using 
the [global](#global) method.
```